test/e2e/fixture/zomes/foo/Cargo.toml

```
@@ -10,4 +10,4 @@ crate-type = [ "cdylib", "rlib" ]
 
 [dependencies]
 serde = "=1.0.104"
-hdk3 = { git = "https://github.com/holochain/holochain", rev = "a1174a5a3f66c47f486be312d38604e49424afaf", package = "hdk3" }
+hdk3 = { git = "https://github.com/holochain/holochain", rev = "3dd53e4ca03c8c0815b4412ea8ebd16f2b19cd47", package = "hdk3" }
```

```suggestion
hdk3 = { git = "https://github.com/holochain/holochain", rev = "041ad6c2fd88c06eace450074621be2bace0eb05", package = "hdk3" }
```

@zippy you have a different REV in here, this is the one you used up above.
These things should stay in sycn

---

whoops